<!doctype html>
<!--[if lt IE 7]><html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en"> <![endif]-->
<!--[if (IE 7)&!(IEMobile)]><html class="no-js lt-ie9 lt-ie8" lang="en"><![endif]-->
<!--[if (IE 8)&!(IEMobile)]><html class="no-js lt-ie9" lang="en"><![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"><!--<![endif]-->
<head>
<meta charset="utf-8">
<title>Microbiome Natural Product Biosynthesis and Synthetic Orphans &#8211; Prophage</title>
<meta name="description" content="Discussion of a recent study that provides new insights into orphan biosynthetic gene clusters, and the implications of those findings for the human microbiome.">
<meta name="keywords" content="Recent Research, Natural Products, Genomics, Bioinformatics">


<!-- Twitter Cards -->
<meta name="twitter:title" content="Microbiome Natural Product Biosynthesis and Synthetic Orphans">
<meta name="twitter:description" content="Discussion of a recent study that provides new insights into orphan biosynthetic gene clusters, and the implications of those findings for the human microbiome.">
<meta name="twitter:site" content="@iprophage">
<meta name="twitter:creator" content="@iprophage">

<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="http://microbiology.github.io/images/prophagelogo.jpg">

<!-- Open Graph -->
<meta property="og:locale" content="en_US">
<meta property="og:type" content="article">
<meta property="og:title" content="Microbiome Natural Product Biosynthesis and Synthetic Orphans">
<meta property="og:description" content="Discussion of a recent study that provides new insights into orphan biosynthetic gene clusters, and the implications of those findings for the human microbiome.">
<meta property="og:url" content="http://microbiology.github.io/blog/bgc-transcript-pnas/">
<meta property="og:site_name" content="Prophage">





<link rel="canonical" href="http://microbiology.github.io/blog/bgc-transcript-pnas/">
<link href="http://microbiology.github.io/feed.xml" type="application/atom+xml" rel="alternate" title="Prophage Feed">

<!-- https://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<!-- For all browsers -->
<link rel="stylesheet" href="http://microbiology.github.io/assets/css/main.css">
<!-- Webfonts -->
<script src="https://use.edgefonts.net/source-sans-pro:n2,i2,n3,i3,n4,i4,n6,i6,n7,i7,n9,i9;source-code-pro:n4,n7;volkhov.js"></script>

<meta http-equiv="cleartype" content="on">

<!-- HTML5 Shiv and Media Query Support -->
<!--[if lt IE 9]>
  <script src="http://microbiology.github.io/assets/js/vendor/html5shiv.min.js"></script>
  <script src="http://microbiology.github.io/assets/js/vendor/respond.min.js"></script>
<![endif]-->

<!-- Modernizr -->
<script src="http://microbiology.github.io/assets/js/vendor/modernizr-2.7.1.custom.min.js"></script>


<!-- MathJax -->
<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>


<!-- Icons -->
<!-- 16x16 -->
<link rel="shortcut icon" href="http://microbiology.github.io/favicon.ico">
<!-- 32x32 -->
<link rel="shortcut icon" href="http://microbiology.github.io/favicon.png">
<!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
<link rel="apple-touch-icon-precomposed" href="http://microbiology.github.io/images/apple-touch-icon-precomposed.png">
<!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
<link rel="apple-touch-icon-precomposed" sizes="72x72" href="http://microbiology.github.io/images/apple-touch-icon-72x72-precomposed.png">
<!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
<link rel="apple-touch-icon-precomposed" sizes="114x114" href="http://microbiology.github.io/images/apple-touch-icon-114x114-precomposed.png">
<!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
<link rel="apple-touch-icon-precomposed" sizes="144x144" href="http://microbiology.github.io/images/apple-touch-icon-144x144-precomposed.png">

</head>

<body id="post">

<div class="navigation-wrapper">
	<nav role="navigation" id="site-nav" class="animated drop">
	    <ul>
      
		    
		    <li><a href="http://microbiology.github.io/" >Home</a></li>
		  
		    
		    <li><a href="http://microbiology.github.io/about/" >About</a></li>
		  
		    
		    <li><a href="http://microbiology.github.io/blog/" >Blog</a></li>
		  
		    
		    <li><a href="http://microbiology.github.io/cv/Hannigan-Geoffrey-D_Curriculum_Vitae.pdf" >CV</a></li>
		  
		    
		    <li><a href="https://github.com/Microbiology" target="_blank">Code</a></li>
		  
		    
		    <li><a href="http://microbiology.github.io/search/" >Search</a></li>
		  
	    </ul>
	</nav>
</div><!-- /.navigation-wrapper -->

<!--[if lt IE 9]><div class="upgrade"><strong><a href="http://whatbrowser.org/">Your browser is quite old!</strong> Why not upgrade to a different browser to better enjoy this site?</a></div><![endif]-->

<header class="masthead">
	<div class="wrap">
      
  		<a href="http://microbiology.github.io/" class="site-logo" rel="home" title="Prophage"><img src="http://microbiology.github.io/images/prophagelogo.jpg" width="200" height="200" alt="Prophage logo" class="animated fadeInDown"></a>
      
      <h1 class="site-title animated fadeIn"><a href="http://microbiology.github.io/">Prophage</a></h1>
		<h2 class="site-description animated fadeIn" itemprop="description">The Blog for Bacteria, Phages, Computers, and Science</h2>
	</div>
</header><!-- /.masthead -->

<div class="js-menu-screen menu-screen"></div>


<div id="main" role="main">
  <article class="hentry">
    
    <div class="entry-wrapper">
      <header class="entry-header">
        <ul class="entry-tags">
          <li><a href="http://microbiology.github.io/tags/#Recent Research" title="Pages tagged Recent Research">Recent Research</a></li><li><a href="http://microbiology.github.io/tags/#Natural Products" title="Pages tagged Natural Products">Natural Products</a></li><li><a href="http://microbiology.github.io/tags/#Genomics" title="Pages tagged Genomics">Genomics</a></li><li><a href="http://microbiology.github.io/tags/#Bioinformatics" title="Pages tagged Bioinformatics">Bioinformatics</a></li>
        </ul>
        
          <h1 class="entry-title">Microbiome Natural Product Biosynthesis and Synthetic Orphans</h1>
        
      </header>
      <footer class="entry-meta">
        
        
          <img src="http://microbiology.github.io/images/bio-photo.jpg" class="bio-photo" alt="Geoffrey Hannigan, PhD bio photo"></a>
        
        <span class="author vcard">By <span class="fn">Geoffrey Hannigan, PhD</span></span>
        <span class="entry-date date published"><time datetime="2018-09-23T21:30:00-04:00"><i class="fa fa-calendar-o"></i> September 23, 2018</time></span>
        
        <span class="entry-comments"><i class="fa fa-comment-o"></i> <a href="#disqus_thread">Comment</a></span>
        <span class="social-share-twitter">
  <a href="https://twitter.com/intent/tweet?hashtags=RecentResearch,NaturalProducts,Genomics,Bioinformatics&amp;text=Microbiome%20Natural%20Product%20Biosynthesis%20and%20Synthetic%20Orphans&amp;url=http://microbiology.github.io/blog/bgc-transcript-pnas/&amp;via=iprophage" title="Share on Twitter" itemprop="Twitter"><i class="fa fa-twitter-square"></i> Tweet</a>
</span>
<span class="social-share-facebook">
  <a href="https://www.facebook.com/sharer/sharer.php?u=http://microbiology.github.io/blog/bgc-transcript-pnas/" title="Share on Facebook" itemprop="Facebook"><i class="fa fa-facebook-square"></i> Like</a>
</span>
<span class="social-share-googleplus">
  <a href="https://plus.google.com/share?url=http://microbiology.github.io/blog/bgc-transcript-pnas/" title="Share on Google Plus" itemprop="GooglePlus"><i class="fa fa-google-plus-square"></i> +1</a>
</span>
<!-- /.social-share -->
        
      </footer>
      <div class="entry-content">
        <p>Wow, it’s been about six months since my last post and a lot has been happening! Between pushing through some papers and working on exciting projects in my new job in pharma, it’s been a busy summer. Unfortunately the blogging took a back seat but I’m excited to hop back onto the computer and hopefully keep the posts more frequent moving forward. So let’s dive in!</p>

<p>One of the exciting aspects of human microbiome research has been the continued improvment of <strong>functional analysis</strong> of the communities through techniques such as metagenomics and metatranscriptomics. One approach to understanding microbiome functionality, which has recently been gaining more traction in the field, has been bacterial <strong>biosynthetic gene clusters (BGCs)</strong>. This focus was especially apparent at recent microbiome/microbiology conferences, including the Beneficial Microbes conference and ASM Microbe conference.</p>

<p>BGCs are co-localized gene clusters within bacterial genomes that together encode the machinery necessary for synthesis of <a href="https://en.wikipedia.org/wiki/Secondary_metabolite">secondary metabolites</a>. These metabolites have been widely used as <a href="https://en.wikipedia.org/wiki/Natural_product">natural product drugs</a> and represent an important area in which the microbiome could be impacting its host. Additionally, bioinformatics techniques have been continuously developed to identify these signature clusters, and those techniques are starting to be utilized with microbiome datasets.</p>

<figure>
  <div style="vertical-align:middle; text-align:center">
  <img src="../../../images/pnas.1714381115fig06.jpg" align="middle" width="60%" />
  <figcaption>One cool concept the paper mentions is the use of Operational Biosynthetic Units for evaluating diversity of the functional potential.</figcaption>
  </div>
</figure>

<p>Although we can use bioinformatics tools to identify BGCs within bacterial genomes, there are still many challenges in the field. One of these major challenges is that a majority of BGCs remain <strong>orphans</strong>, which means they have yet to be linked to their secondary metabolite products. A recent paper by Amos <em>et al</em>, published in PNAS and titled <a href="https://www.ncbi.nlm.nih.gov/pubmed/29229817">Comparative Transcriptomics as a Guide to Natural Product Discovery and Biosynthetic Gene Cluster Functionality</a>, used transcriptomic and metabolomic approaches to provide important insights into orphan BGCs. This week will review some of the highlights from this study and briefly discuss the impact on the human microbiome field.</p>

<p>The core of the paper focused on four strains of the bacterium actinomycete <em>Salinospora</em>, marine bacteria which contain high numbers of BGCs. The group started by addressing the question of whether orphan BGC products are undetectable due to transcriptional silence (inactivity) in the laboratory. By performing global transcriptomic analyses (identification of which BGCs are active at the RNA expression level) and pairing it with metabolomic analysis (identification of metabolites, including BGC-produced secondary metabolites) at both stationary and growth phases, the group revealed an interesting phenomenon. While many BGCs were active at the gene transcript level (and therefore not silent), the BGC products (secondary metabolites) were still not readily identifiable. Although some orphan BGCs are the result of transcriptional inactivity (gene silence), this result highlights the important limitations of the other detection techniques such as LC/MS techniques, and the importance of understanding the transcriptional activity of the system being studied.</p>

<figure>
  <div style="vertical-align:middle; text-align:center">
  <img src="../../../images/pnas.1714381115fig03.jpg" align="middle" width="60%" />
  <figcaption>Comparison of bacterial strain genomes and their transcription profiles revealed other genomic elements whose presence influences BGC expression.</figcaption>
  </div>
</figure>

<p>The group followed up on this finding by investigating why some BGCs were differentially expressed between the bacterial strains. In other words, why was the same BGC transcriptionally active in one strain and not another highly similar strain? In one notable example, the group found that one orphan BGC was expressed when it included an activator (araC) upstream of the core BGC operon, and was silent when the activator was replaced with a different gene. This is an especially important result because it highlights our potential ability to use additional genes, such as upstream transcriptional activators, to predict which highly similar BGCs will be expressed in related bacterial genomes. This would be a valuable bioinformatics tool for informing the study of the human microbiome, as well as natural product drug discovery.</p>

<p>Overall this is a very interesting paper with a lot of insights into BGC activity and regulation. But why is this important from a human microbiome perspective? As a lot of groups in the field have already identified, BGC identification techniques from metagenomic datasets of the human microbiome offer an opportunity to identify largely unrecognized functional potential of the communities. The approach also offers implications for natural product discovery from such microbiome datasets. This paper further highlights the importance of integrating transcriptomic and metabolomic analyses into such studies, so that a more accurate observation of the community activity can be achieved.</p>


        
          <div id="disqus_thread"></div><!-- /#disqus_thread -->
          
<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
    var disqus_shortname = 'http-microbiology-github-io'; // required: replace example with your forum shortname

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = 'https://' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function () {
        var s = document.createElement('script'); s.async = true;
        s.type = 'text/javascript';
        s.src = 'https://' + disqus_shortname + '.disqus.com/count.js';
        (document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s);
    }());
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<a href="https://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>

        
      </div><!-- /.entry-content -->
    </div><!-- /.entry-wrapper -->
    <nav class="pagination" role="navigation">
      
        <a href="http://microbiology.github.io/blog/com-bio-tech/" class="btn" title="Three Technologies With a Future In Computational Biology">Previous</a>
      
      
        <a href="http://microbiology.github.io/blog/recent-crc-virome/" class="btn" title="Recent Publication: The Colorectal Cancer Virome">Next</a>
      
    </nav><!-- /.pagination -->
  </article>
</div><!-- /#main -->

<div class="footer-wrapper">
  <footer role="contentinfo" class="entry-wrapper">
    

<span>&copy; 2023 Geoffrey Hannigan, PhD. Powered by <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a> using the <a href="https://mademistakes.com/work/so-simple-jekyll-theme/" rel="nofollow">So Simple Theme</a>.</span>
<div class="social-icons">
	<a href="https://twitter.com/iprophage" title="Geoffrey Hannigan, PhD on Twitter" target="_blank"><i class="fa fa-twitter-square fa-2x"></i></a>
	
	
	
	
	
	
	<a href="https://github.com/Microbiology" title="Geoffrey Hannigan, PhD on Github" target="_blank"><i class="fa fa-github-square fa-2x"></i></a>
	
  
	
  <a href="http://microbiology.github.io/feed.xml" title="Atom/RSS feed"><i class="fa fa-rss-square fa-2x"></i></a>
</div><!-- /.social-icons -->

  </footer>
</div><!-- /.footer-wrapper -->

<script type="text/javascript">
  var BASE_URL = 'http://microbiology.github.io';
</script>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
<script>window.jQuery || document.write('<script src="http://microbiology.github.io/assets/js/vendor/jquery-1.9.1.min.js"><\/script>')</script>
<script src="http://microbiology.github.io/assets/js/scripts.min.js"></script>


<!-- Asynchronous Google Analytics snippet -->
<script>
  var _gaq = _gaq || [];
  var pluginUrl =
 'https://www.google-analytics.com/plugins/ga/inpage_linkid.js';
  _gaq.push(['_require', 'inpage_linkid', pluginUrl]);
  _gaq.push(['_setAccount', 'UA-73732103-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = 'https://stats.g.doubleclick.net/dc.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>



</body>
</html>
